src/sentry_options.c

```diff
@@ -32,6 +32,7 @@ sentry_options_new(void)
     opts->environment = sentry__string_clone(getenv("SENTRY_ENVIRONMENT"));
 #endif
     opts->user_consent = SENTRY_USER_CONSENT_UNKNOWN;
+    opts->auto_session_tracking = true;
```

Comment:
Shouldn't this be false on Android, or would they just set `false` in the Android SDK wrapper?

Reply:
good catch!

Reply:
Thinking about this again, setting `false` in the wrapper is probably the best solution. That way, it's still enabled when running the native SDK alone.

Reply:
I agree. Heads up @marandaneto

Reply:
ok, we can disable on SDK init, thanks for the heads up